_src/container-attrs.md

```diff
@@ -52,6 +52,14 @@
   Use the untagged enum representation for this enum. See [enum
   representations](enum-representations.md) for details on this representation.
 
+- ##### `#[serde(variant_identifier)]` {#variant--identifier}
+
+  Use an identifier representation for this enum. This can only be applied to enums that are C-like (containing only unit variants), and forces them to always be represented as strings, regardless of the underlying data format's representation of enums.
+
+- ##### `#[serde(field_identifier)]` {#field--identifier}
+
+  Identical to [`variant_identifier`](#variant--identifier), but also allows for the last variant to be a newtype variant, which will be used if none of the other variants match (similar to [`#[serde(other)]`](../variant-attrs.html#other)). Like `variant_identifier`, this forces the enum to always be represented as a string, regardless of the underlying data format's representation of enums.
```

Comment:
Could we have an example here?

It doesn't seem to work based on the description:

```rust
#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
#[serde(rename_all = "camelCase", field_identifier)]
pub enum Type {
    NixosModule,
    Derivation,
    App,
    Unknown(String),
}
```

<img width="434" alt="image" src="https://github.com/serde-rs/serde-rs.github.io/assets/3998/f71cd21f-dfdf-42f8-b950-4cd7ccfba67c">

Reply:
Okay, so when using this condition, `Serialize` cannot be derive automatically. That should be mentioned, although I wonder why this limitation even exists.

Reply:
Oh, that's a good catch. Probably it was only *intended* to support the intermediary enum types used during deserialization, so there was never a need to add serialize support. That's something I can look into adding in parallel with this docs PR.